bo/messlokation_test.go

```diff
@@ -32,7 +33,7 @@ func (s *Suite) Test_Messlokation_Deserialization() {
 		},
 		MesslokationsId:              "DE0000011111222223333344444555556",
 		Sparte:                       sparte.STROM,
-		NetzebeneMessung:             netzebene.MD,
+		NetzebeneMessung:             internal.Ptr(netzebene.MD),
```

Comment:
what does the `internal.Ptr()`?

Reply:
internal.Ptr(netzebene.MD) = &netzebene.MD

aber in kompliziert, weil go so einfach ist.

Reply:
I see, trivial knowledge